src/dc_receive_imf.rs

```diff
@@ -1837,6 +1857,44 @@ async fn create_or_lookup_mailinglist(
     }
 }
 
+async fn apply_mailinglist_changes(
+    context: &Context,
+    mime_parser: &MimeMessage,
+    chat_id: ChatId,
+) -> Result<()> {
+    let mut chat = Chat::load_from_db(context, chat_id).await?;
+    if chat.typ != Chattype::Mailinglist {
+        return Ok(());
+    }
+    let listid = &chat.grpid;
+
+    if let Some(list_post) = &mime_parser.list_post {
+        let (contact_id, _) =
+            Contact::add_or_lookup(context, "", list_post, Origin::Hidden).await?;
+        let mut contact = Contact::load_from_db(context, contact_id).await?;
+        if contact.param.get(Param::ListPost) != Some(listid) {
```

Comment:
Maybe actually create a `Param::ListId` for `List-Post` contacts. This is hard to read unless you know that `ListPost` param contains `ListId` to save one param letter.

Reply:
fixed in ff7b8645